Gemfile

```
@@ -23,7 +23,7 @@ group :test do
   gem "nokogiri", "~> 1.7"
   gem "rspec"
   gem "rspec-mocks"
-  gem "rubocop", "~> 1.0"
+  gem "rubocop", "~> 1.8"
```

To pin version to a minor series, you should include the patch version as well:

```suggestion
  gem "rubocop", "~> 1.8.1"
```

---

Right. Thanks for catching that. Changed.